File: programmers/python/Lv0/조건문자열.py
def solution(ineq, eq, n, m):
    if ineq == ">":
        answer1 = n > m
    elif ineq == "<":
        answer1 = n < m
    if eq == "=":
        answer2 = n == m
    elif eq == "!":
        answer2 = 0
    answer = int(answer1 or answer2)
    return answer

File: programmers/java/Lv0/조건문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        Boolean answer;
        if (ineq.equals("<")) {
            answer = n < m;
        }
        else {
            answer = n > m;
        };
        
        if (eq.equals("=")) {
            answer = answer || (n == m);
        };
        
        return (answer) ? 1:0;
    }
}